opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp

```diff
@@ -565,7 +565,9 @@ class Well {
     bool segmented_density_calculation() const { return true; }
     double alq_value() const;
     double temperature() const;
+    bool hasInjected( ) const;
     bool hasProduced( ) const;
+    bool updateHasInjected( );
     bool updateHasProduced();
     bool cmp_structure(const Well& other) const;
     bool operator==(const Well& data) const;
```

Comment:
this needs updating.

Reply:
Thanks - I have updated the `operator==` - I assume that was what you meant?

Reply:
yes, exactly.